aip/general/0111.md

```diff
@@ -0,0 +1,90 @@
+---
+id: 111
+state: draft
+created: 2023-06-10
+placement:
+  category: api-concepts
+  order: 50
+---
+
+# Planes
+
+Resources and methods on an API can be divided into the *plane* that they reside
+or perform operations upon. For the context of APIs, the following planes are
+defined:
+
+- The management plane: a uniform, resource-oriented API that primarily
```

Comment:
We probably need to at least mention the control plane here, even if it may not be distinct from the management plane, since that term is likely more familiar. 

Also, some people may consider the IaC layer the management plane and the resource APIs to be the control plane.

Reply:
That begs the question: do we want to introduce a term like the management plane? It's a bit easier to define operations in terms of their impact on the control plane vs data-plane.

Although I guess this wouldn't capture that nuance that sometimes an operation may want to operate on both (e.g. control plane provisions a database and immediately populates it with a table or permissions).